src/nginxconfig/i18n/zh-tw/templates/global_sections/docker.js

```diff
@@ -32,10 +32,10 @@ const dockerfile = 'Dockerfile';
 export default {
     docker,
     dockerfile,
-    dockerCompose: `${docker} Compose`, // TODO: translate
-    applyDockerTweaks: `Apply ${docker} tweaks`, // TODO: translate
-    applyDockerTweaksForNginx: `Apply configuration tweaks for running ${common.nginx} with ${docker}`, // TODO: translate
-    applyDockerTweaksExplainer: `Updates the ${common.nginx} user to be <code class="slim">nginx</code> and the pid to <code class="slim">/var/run/nginx.pid</code>`, // TODO: translate
+    dockerCompose: `${docker} 編排服務`,
+    applyDockerTweaks: `應用${docker}調整`,
+    applyDockerTweaksForNginx: `為使用${docker}運行${common.nginx}進行配寘調整。`,
+    applyDockerTweaksExplainer: `將${common.nginx}用戶更新為<code class=“slim”>nginx</code>，將pid更新為<code class=“slim”>/var/run/nginx.pid</code>`,
```

Comment:
```suggestion
    applyDockerTweaksExplainer: `將${common.nginx}用戶更新為<code class="slim">nginx</code>，將pid更新為<code class="slim">/var/run/nginx.pid</code>`,
```
Needs normal quotes for the HTML